src/index.html

```diff
@@ -11,7 +11,160 @@
+            name="name"
+            autocomplete="off"
+            required
+          >
+        </label>
+        <label class="form__field">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            value="12"
+            min="1"
+            max="100"
+            required
+          >
+        </label>
+        <label class="form__field">
+          Full date of birth:
+          <input
+            type="date"
+            name="dateOfBirth"
+            required
+          >
+        </label>
+        <label>
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="terms"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset__form">
+        <legend>Registration</legend>
+        <label class="form__field">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            required
+            autocomplete="off"
+          >
+        </label>
+        <label>
+          Password:
+          <input
+            type="password"
+            name="password"
+            minlength="5"
+            maxlength="32"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset__form">
+        <legend>An interesting fact about you!</legend>
+        <div class="form__field">
+          Do you love cats?
+          <label>
+            <input
+              type="radio"
+              name="isLoveCats"
+              value="yes"
+            >
+            Yes
+          </label>
+          <label>
+            <input
+              type="radio"
+              name="isLoveCats"
+              value="no"
+            >
+            No
+          </label>
+        </div>
+        <label class="form__field">
+          What is your favorite color?
+          <input
+            type="color"
+            name="color"
+            value="#000"
+          >
+        </label>
+        <label class="form__field">
+          What time do you go to bed?
+          <input
+            type="time"
+            name="timeGoToBed"
+          >
+        </label>
+        <div class="form__field">
+          What are your favorite brand of cars?
+          <select name="cars" multiple>
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+        </div>
+        <label>
+          How do you rate our work?
+          <input
+            type="range"
+            name="rate"
+            min="1"
+            max="10"
+            step="1"
+            value="0"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset__form">
+        <legend>Additional info</legend>
+        <label class="form__field">
+          Comments:
```

Comment:
checklist p.8  - Check the textarea behaviour - when you click on the textarea, the coursor should appear on the beginning of the input.

Reply:
I have changed it.